doc/Language/typesystem.pod6

```diff
@@ -258,6 +258,49 @@ you must use private attribute notation instead.
     C.new.say; C.new('answer').say;
     # OUTPUT: «C.new(attr => 42)␤
     #          C.new(attr => "answer")␤»
+    
+=head4 submethod TWEAK
+
+The L<submethod|/type/Submethod> C<TWEAK> is (indirectly) called by
+L<.bless|/type/Mu#method_bless>. It
+is meant to set private and public attributes of a class after the BUILD submethod has been called. The default
```

Comment:
meant to compute attributes that can't be known at build time. You can set them in the `BUILD` phaser.

Reply:
submethod BUILD and BUILD phaser are different. The first is for object instantiation and can occur many times. The second is for code compilation of a program, which probably should only occur once per run.

Reply:
Well, they're "phasers" when building an object. Changed it to submethod, the main point still holds; you can set private and public attributes in BUILD.

Reply:
So far as I can recall, the documents have never referred to submethod BUILD as a phaser. I seem to remember playing around with submethod BUILD before Phasers were available. And I certainly remember not being able to do things with BUILD alone that can be done easily with TWEAK.